### PR TITLE
Added CD

### LINKED ISSUE #11: Add CD to repo
It's too hard all the time update prod, so it would be nice to add CD on updating master branch.

### PR DESCRIPTION
**Pull request by bond95**
Closes: https://github.com/bond95/ToxicBot/issues/11

### CODE REVIEW

File: deploy,sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+_remote="ls.backup"
+_user="vivek"
```
Comment (DevAlone) [inline deploy,sh]: Why is it needed?

File: deploy,sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+echo "Local system name: $HOSTNAME"
+echo "Local date and time: $(date)"
+
+echo
+echo "*** Running commands on remote host named $_remote ***"
+echo
+ssh  "$DEPLOY_LOGIN@$DEPLOY_HOST"  <<'EOL'
+	kill $(ps aux | grep 'toxic_bot' | awk '{print $2}')
```
Comment (DevAlone) [inline deploy,sh:4]: I guess it's the same as pkill, not sure, but it doesn't matter

File: deploy,sh (hunk 1 above)
Comment (bond95) [reply deploy,sh]: because i didn't delete it

**Comment by DevAlone** [comment]
Looks fine(few moments I didn't fully realize above). As a future improvement, I'd suggest creating a systemd unit since current approach can lead to the bot being unavailable because of an error.

**Comment by bond95** [comment]
Yup

**Pull request merged**